x/blob/test/decode_blob_tx_test.go

```diff
@@ -0,0 +1,111 @@
+package test
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/pkg/blob"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+)
+
+type BlockResponse struct {
+	Block Block `json:"block"`
+}
+
+type Block struct {
+	Data BlockData `json:"data"`
+}
+
+type BlockData struct {
+	Txs []string `json:"txs"`
+}
```

Comment:
Why not use the types provided in cometbft?

Reply:
Good point, can do.